packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts

```diff
@@ -267,6 +269,8 @@ export function getViemNetwork(networkName: EVMBlockchainIncludingTestnet) {
             return astarZkEVM;
         case "optimism-sepolia":
             return optimismSepolia;
+        case "arbitrum-sepolia":
+            return arbitrumSepolia;
```

Comment:
nit: can we move this to line 258 to group the arbitrum ones together?

Reply:
I grouped all of them. LMK if looks good